stacks/PickupStack.ts

```diff
@@ -117,8 +141,33 @@ export function PickupStack ({ app, stack }: StackContext): void {
     basicApi.bucket.cdk.bucket.grantReadWrite(service.taskDefinition.taskRole)
     basicApi.dynamoDbTable.cdk.table.grantReadWriteData(service.taskDefinition.taskRole)
     basicApi.queue.cdk.queue.grantConsumeMessages(service.taskDefinition.taskRole)
+
+    if (process.env.USE_VALIDATION === 'VALIDATE') {
```

Comment:
Which values `process.env.USE_VALIDATION` can be? Looks like a boolean

Reply:
Is a boolean, but I don't manage the ENV, I need to be sure that's enabled only on the `VALIDATE` value